Create a blog post about the sprint 156 (#446)
openSUSE/obs-landing: _posts/development/2023-12-21-request-reports-and-comment-reports.md

---
layout: post

> title: Request reporting and easier reporting of users from comments

Reviewer: Please "titleize" the titles.

```suggestion
title: Request Reporting and Easier Reporting of Users from Comments
```

But we could even simplify it like: "New and Improved Ways to Report" or "Expanded and Improved Reporting"... What do you think?

category: development
image: images/posts/sprint_156/page_image_user_report.png
---

> In order to be effective, the content moderation feature has to be featureful and streamlined, so now we expanded it by a new kind of reportable and an easier way to report creators of comments.

Reviewer: ```suggestion
In order to be effective, the content moderation feature has to be featureful and streamlined, so now we expanded it with a new kind of reportable and an easier way to report creators of comments.
```

??

*Content Moderation* is part of the [beta program](/2018/10/04/the-beta-program/).

{% include partials/_series-of-posts-about-content-moderation.md %}

> ### Reporting requests

Reviewer: ```suggestion
### Reporting Requests
```

> Users can create various kinds of content in OBS, which means that spammers can do it too. We expanded the reporting feature by adding Request into the mix.

Reviewer 2: ```suggestion
Users can create various kinds of content in OBS, which means that malicious people can do it too. We expanded the reporting feature by adding Request into the mix.
```

<figure>

>   <img src="/images/posts/sprint_156/request_report.png" alt="Screenshot of the dialog for reporting a request" />

Reviewer: It might be helpful that this screenshot includes the link to "Report Request" on the left-side menu. So people know where to look it for (it might be highlighted somehow).

  <figcaption>The dialog for reporting a request</figcaption>
</figure>

> ### Reporting a comment creator with the comment

Reviewer: ```suggestion
### Reporting a Comment Creator with the Comment
```

> Very often when reporting a comment, you might want the moderators to also have a deeper look into the commenter. We expanded the report dialog to include that option as a simple checkbox.

Reviewer 2: ```suggestion
When reporting a comment, you likely want the moderators to also have a deeper look into the commenter. We expanded the report dialog to include that option as a simple checkbox.
```
